src/tagging/sqlite/tagging.py

```diff
@@ -0,0 +1,971 @@
+# -*- coding: utf-8 -*- {{{
+# ===----------------------------------------------------------------------===
+#
+#                 Installable Component of Eclipse VOLTTRON
+#
+# ===----------------------------------------------------------------------===
+#
+# Copyright 2022 Battelle Memorial Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy
+# of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# ===----------------------------------------------------------------------===
+# }}}
+import contextlib
+import csv
+import errno
+import logging
+import os
+import re
+import sqlite3
+import sys
+from collections import OrderedDict
+
+from gevent.local import local
+
+from tagging.base.base_tagging import BaseTaggingAgent
+from volttron import utils
+from volttron.client.messaging.health import STATUS_BAD, Status
+from volttron.utils import ClientContext
+
+__version__ = "1.1"
```

Comment:
so this should match the poetry at least.

Reply:
updated. but how do we keep up when we push to main?

Reply:
a couple of different ways...we have a shell script like there is in the multirepo that does a search/replace over the repo when it's built.

Or we do it manually?  BLEH...we can also make use of the one in the pyproject.toml.  I think